### DISCUSSION TITLE
counterDeliveredToday

### DISCUSSION THREAD

**Discussion by Peter-Bos** [addiejanssen/domoticz-solaredge-modbustcp-plugin#75] [Q&A]
The value of counterDeliveredToday in Domoticz is not set to 0 at midnight, but keeps the value of the day before until the first production of the day. Is this a feature of the plugin or is it a bug in Domoticz?

**Comment by addiejanssen** [comment]
It's probably a combination of topics. That counter must be a Domoticz thingie - I have never "seen" it before and it is not used/updated by the plugin. The plugin will not deliver updates if there is nothing to retrieve from the inverter (for example when it is waiting for the panels to start producing energy again). So your observation makes sense.
